tests/test_api.py

```diff
@@ -104,7 +83,8 @@ def test_load_survey(self, bliss_client, monkeypatch):
             f"{bliss_client.cwd}/{download_dir}/94/1/12/psField-000094-1-0012.fits"
         ).exists()
 
-    def test_predict_sdss_default_rcf(self, bliss_client, weight_save_path, cfg):
-        paths = {"sdss": cfg.paths.sdss, "decals": cfg.paths.decals}
-        bliss_client.predict_sdss(weight_save_path=weight_save_path, paths=paths)
+    def test_predict_sdss_default_rcf(self, cfg, bliss_client, monkeypatch):
+        monkeypatch.setattr(api, "_predict_sdss", mock_predict_sdss)
+        # cached predict data stored at cfg.paths.data, copied to temp dir in mock_predict_sdss
+        bliss_client.predict_sdss("test_path", paths={"data": cfg.paths.data})
```

Comment:
I am conflicted about overriding `paths.data` because it is not something an end user would typically do.

Reply:
I'm only overriding it so the correct path gets passed in the config to `mock_predict_sdss`, which uses it to load the cached predictions and return them. The alternative would be to hardcode it in `mock_predict_sdss` - not sure if that would be better

Reply:
Just hardcoded it for simplicity